### PR TITLE
Support string array for configuration keys

### PR DESCRIPTION
**Pull request by vladlemberg**
### Description:

Please refer https://github.com/osmlab/atlas-checks/issues/492

### Potential Impact:

Ability to make complex description for Atlas Checks with backward compatibility.

Example:
"description": ["sentense1.",
                         "sentence2.",
                         etc..]

### Unit Test Approach:

N/A

### Test Results:

Backward compatibility tested.

### CODE REVIEW

**Review by atiannicelli — DISMISSED** [review]
LGTM!

File: src/main/java/org/openstreetmap/atlas/checks/maproulette/serializer/ChallengeDeserializer.java [hunk 1]
```diff
@@ -84,6 +87,27 @@ private String getStringValue(final JsonObject object, final String key,
...
         {
+            if (object.get(key) instanceof JsonArray)
+            {
+                try
+                {
+                    final StringBuilder stringBuilder = new StringBuilder();
+                    final Gson googleJson = new Gson();
+                    final ArrayList<String> jsonObjList = googleJson
+                            .fromJson(object.get(key).getAsJsonArray(), ArrayList.class);
+                    for (final Object obj : jsonObjList)
```
Comment (Bentleysb) [inline src/main/java/org/openstreetmap/atlas/checks/maproulette/serializer/ChallengeDeserializer.java:98]: Please don't use abbreviations.

File: src/main/java/org/openstreetmap/atlas/checks/maproulette/serializer/ChallengeDeserializer.java [hunk 2]
```diff
@@ -84,6 +87,27 @@ private String getStringValue(final JsonObject object, final String key,
...
+                    {
+                        stringBuilder.append(obj);
+                    }
+                    return stringBuilder.toString();
+                }
+                catch (final Exception e)
```
Comment (Bentleysb) [inline src/main/java/org/openstreetmap/atlas/checks/maproulette/serializer/ChallengeDeserializer.java:104]: Please don't use single letter variable names.

**Review by Bentleysb — COMMENTED** [review]
Looks great. Just need a couple of variable name changes.

**Review by Bentleysb — APPROVED** [review]

**Comment by ameliaewatts** [comment]
> LGTM!

@atiannicelli This is good to go, yes? Can you "approve" it? 😄

**Comment by ameliaewatts** [comment]
@sayas01 Can we get your review?

**Review by sayas01 — APPROVED** [review]
LGTM!

**Pull request merged**